app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -13,6 +13,8 @@ class ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job < Job
   ERRCODE_POD_NOTFOUND = 404
   IMAGE_INSPECTOR_SA = 'inspector-admin'
   INSPECTOR_ADMIN_SECRET_PATH = '/var/run/secrets/kubernetes.io/inspector-admin-secret-'
+  IMAGE_INSPECTOR_ATTRIBUTE_SECTION = 'cluster_settings'
```

Comment:
@enoodle you may now want to reconsider the name of `IMAGE_INSPECTOR_ATTRIBUTE_SECTION`.